### PR TITLE
Many fixes

### PR DESCRIPTION
**Pull request by emmanuelm41**
## Context (issues, jira)
Filecoin integration

## Description / Usage
- add derivation mode on account ids
- increase timeout on specs
- fix mutation flag on bot cli cmd

## Expectations

- [ ] **Test coverage: The changes of this PR are covered by test.** Unit test were added with mocks when depending on a backend/device.
- [ ] **No impact: The changes of this PR have ZERO impact on the userland.** Meaning, we can use these changes without modifying LLD/LLM at all. It will be a "noop" and the maintainers will be able to bump it without changing anything.

### CODE REVIEW

File: src/families/filecoin/specs.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ const filecoinSpecs: AppSpec<Transaction> = {
     appName: "Filecoin",
   },
 
-  testTimeout: 2 * 60 * 1000,
+  testTimeout: 5 * 60 * 1000,
```
Comment (haammar-ledger) [inline src/families/filecoin/specs.ts:19]: This seems like a lot, users would need to wait 5min to see their transactions confirmed?
Any chance confirmed transactions get seen sooner in the future?

File: src/bot/index.ts [hunk 2]
```diff
@@ -77,7 +77,7 @@ export async function bot({ currency, family, mutation }: Arg = {}) {
         if (mutation) {
           spec = {
             ...spec,
-            mutations: spec.mutation.filter((m) =>
+            mutations: spec.mutations.filter((m) =>
```
Comment (haammar-ledger) [inline src/bot/index.ts:80]: Nice catch @emmanuelm41!
cc @gre

**Review by ghost — APPROVED** [review]

File: src/families/filecoin/specs.ts (hunk 1 above)
Comment (emmanuelm41) [reply src/families/filecoin/specs.ts:19]: In practice, it took less than 3 minutes. A little more than 2 minutes. I think in the future we will improve it tuning configuration parameters to make it faster.

File: src/bot/index.ts (hunk 2 above)
Comment (emmanuelm41) [reply src/bot/index.ts:80]: I tried to use it and it was failing haha

**Pull request merged**